packages/frontier-plugins/plugin-shared/src/errors/index.ts

```diff
@@ -0,0 +1,13 @@
+import chalk from 'chalk';
+
+export function validityFailed(commandType: string): Promise<void> {
+    // block command unless being run within an rdvue project
+    throw new Error(
+    JSON.stringify({
+        code: 'project-invalid',
+        message: `${commandType} command must be run in an existing ${chalk.yellow('rdvue')} project`,
```

Comment:
The use of rdvue here, makes this single use, can we make this generic to accept options so we can throw errors for different project types and for different reasons.